app/app.js

```diff
@@ -0,0 +1,30 @@
+// Define the AdminUI module
+var adminUI = angular.module('adminUI', []);
+
+// Define the PlaylistController on the adminUI module
+adminUI.controller('PlaylistController', function PlaylistController($scope) {
```

Comment:
I think in the future we are going to want to use ui-router to create views and break up the controllers into other files. but this is good for the mean time!